enterprise/server/githubapp/githubapp.go

```diff
@@ -58,63 +58,80 @@ import (
 )
 
 var (
-	enabled       = flag.Bool("github.app.enabled", false, "Whether to enable the BuildBuddy GitHub app server.")
-	clientID      = flag.String("github.app.client_id", "", "GitHub app OAuth client ID.")
-	clientSecret  = flag.String("github.app.client_secret", "", "GitHub app OAuth client secret.", flag.Secret)
-	appID         = flag.String("github.app.id", "", "GitHub app ID.")
-	publicLink    = flag.String("github.app.public_link", "", "GitHub app installation URL.")
-	privateKey    = flag.String("github.app.private_key", "", "GitHub app private key.", flag.Secret)
-	webhookSecret = flag.String("github.app.webhook_secret", "", "GitHub app webhook secret used to verify that webhook payload contents were sent by GitHub.", flag.Secret)
-
-	enableReviewMutates = flag.Bool("github.app.review_mutates_enabled", false, "Perform mutations of PRs via the GitHub API.")
+	readWriteAppEnabled       = flag.Bool("github.app.enabled", false, "Whether to enable the read-write BuildBuddy GitHub app server.")
+	readWriteAppClientID      = flag.String("github.app.client_id", "", "GitHub app OAuth client ID.")
+	readWriteAppClientSecret  = flag.String("github.app.client_secret", "", "GitHub app OAuth client secret.", flag.Secret)
+	readWriteAppID            = flag.String("github.app.id", "", "GitHub app ID.")
+	readWriteAppPublicLink    = flag.String("github.app.public_link", "", "GitHub app installation URL.")
+	readWriteAppPrivateKey    = flag.String("github.app.private_key", "", "GitHub app private key.", flag.Secret)
+	readWriteAppWebhookSecret = flag.String("github.app.webhook_secret", "", "GitHub app webhook secret used to verify that webhook payload contents were sent by GitHub.", flag.Secret)
+	enableReviewMutates       = flag.Bool("github.app.review_mutates_enabled", false, "Perform mutations of PRs via the GitHub API.")
```

Comment:
would it work to use `flag.Struct` to avoid duplicating all of the config flags, and allow sharing the validation logic?

Reply:
Good idea. Unfortunately I tried, and the `flag.Struct` parser doesn't like the private keys, which have new line characters 

I downloaded the private keys directly from Github. They look like
-----BEGIN RSA PRIVATE KEY-----
...
-----END RSA PRIVATE KEY-----